contributors.txt

```diff
@@ -152,3 +152,7 @@ YYYY/MM/DD, github id, Full name, email
 2021/01/24, bcolaco, Bruno Colaço, bcolaco[@)gmail.com
 2021/01/25, guybark, Guy Barker, guybark(at)microsoft.com
 2021/02/10, mabako, Marcus Bauer, mabako(at)gmail.com
+2021/03/11, AntonKedrov, Anton Kedrov, a-k(at)gmx.com
+2021/03/16, JuPrgn, Julien Peyregne, julienprgn(at)gmail.com
+2021/03/20, nyankoframe, Damit Senanayake, damit(at)outlook.com
+2021/03/31, darkcamper, Rubén Ruiz, darkcamper(at)gmail.com
```

Comment:
This is incorrect, please only add yourself

Reply:
Thank you for the clarification. Updated in latest revision.